src/views/docs/en/guides/developer-experience/local-development.md

```diff
@@ -4,112 +4,103 @@ category: Developer experience
+> 🏁 To set up Architect locally follow the [quickstart guide](/docs/en/guides/get-started/quickstart).
 
-```bash
+Preview a project running locally in a web browser by starting Architect's Sandbox:
+
+```console
 cd myproject
 arc sandbox
 ```
->  `arc sandbox` kicks up a local web server with all the resources defined in your `app.arc` file
-## Testing
+
+>  [`arc sandbox`](/docs/en/reference/cli/sandbox) spins up a local web server with all the resources defined in manifest, emulating a suite of AWS features.
 
 Checkout [a complete example project for working locally](https://github.com/architect-examples/arc-example-working-locally).
 
-### Testing `@http` functions
+## Creating a new resource
+
+You can create a new resource (HTTP route, event, scheduled task, etc.) to an existing project by simply adding a line to the [Architect manifest file](/docs/en/guides/get-started/project-layout#manifest-file-format-overview). The following example will provision a new "hit counter" event.
+
+> 👉 Note: this example uses Node.js conventions but the process is similar for the [Ruby](/docs/en/reference/runtime/ruby) (bundler) and [Python](/docs/en/reference/runtime/python) (pip) runtimes.
+
+### Update Architect's configuration
+
+To create a new event, add an entry to your manifest's [`@events` pragma](/docs/en/reference/app.arc/events).
+
+Sample `app.arc` file with a new `hit-counter` event:
+
+```arc
+# app.arc
+@app
+my-app
+
+@events
+hit-counter
+```
+
+> 💡 If the pragma does not already exist, add "@events" on a new line.
+
+### Scaffold the new resource with `arc init`
+
+The Architect CLI [`init` command](/docs/en/reference/cli/init) can be used to create scaffolding for the new event. From the project root:
 
-Use `@architect/sandbox` as a module to test `@http` functions with Node:
+```console
+arc init
+```
+
+> 👀 Notice that `arc` finds your updated Architect manifest and creates new project files for the hit-counter event. `arc init` can be run as needed while developing your application.
 
-```javascript
-const test = require('tape')
-const tiny = require('tiny-json-http')
-const sandbox = require('@architect/sandbox')
+The following structure was added your project:
+<!-- unsure if this diagram is necessary -->
+```
+.
+├── src
+│   └── events
+│       └── hit-counter
+│           ├── index.js
+│           └── config.arc
+│
+└── app.arc
+```
 
-test('setup', async t=> {
-  t.plan(1)
-  await sandbox.start()
-  t.ok(true, 'sandbox started on http://localhost:3333')
-})
+### Add dependencies (optional)
 
-test('get /', async t=> {
-  t.plan(1)
-  let result = await tiny.get({ url: 'http://localhost:3333' })
-  t.ok(result, 'got 200 response')
-})
+Your new event may require some dependencies. For this example, [@architect/functions](https://github.com/architect/functions) will be helpful to handle the event subscription. Dependencies can be installed directly to the `hit-counter` directory:
```

Comment:
Maybe we can link to the [dependency management guide](https://arc.codes/docs/en/guides/developer-experience/dependency-management) in here somehow to note the two approaches to dependency management? 
Perhaps something like:

```suggestion
Your new event may require some dependencies. For this example, [the @architect/functions runtime helper library](../../reference/runtime/node.js) will be helpful to handle the event subscription. [Dependencies in node.js can be managed](dependency-management) either globally from the root `package.json`, or managed via separate `package.json` files in each function's directory. In this example, let's manage them directly in the event's `src/events/hit-counter` directory:
```